.env.example

```diff
@@ -14,8 +14,7 @@ KODI_PORT=
 KODI_USERNAME=
 KODI_PASSWORD=
 
-SKILL_APPID=
-SKILL_VERIFY_CERT=
```

Comment:
I also don't see where you explicitly enable cert verification?  From the documentation (and I've verified it in execution too), Flask-Ask enables it by default.

Wasn't sure if we wanted a way to disable it, though.  Personally I'd never turn it off, so it's no big deal to me if you'd rather just let Flask-Ask handle it.